### DISCUSSION TITLE
Welcome to ios-toast-alerts Discussions!

### DISCUSSION THREAD

**Discussion by eduardo22i** [estampworld/ios-toast-alerts#6] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by eerwaz9** [comment]
Suppose STACK has 6 memory cells as its maximum size and initially its top= -1. Find the output of the following pseudo code:

**Reply by eduardo22i** [reply]
Where is the pseudo?
